prov/gni/src/gnix_vc.c

```diff
@@ -226,6 +235,8 @@ static void __gnix_vc_unpack_resp(char *rbuf,
 	memcpy(resp_smsg_attr, cptr, sizeof(gni_smsg_attr_t));
 	cptr += sizeof(gni_smsg_attr_t);
 	memcpy(resp_irq_cq_mhndl, cptr, sizeof(gni_mem_handle_t));
+	cptr += sizeof(gni_mem_handle_t);
+	memcpy(caps, cptr, sizeof(uint64_t));
```

Comment:
Unpacking here, but if we feel we need a length check above, we should put one here too.